src/libraries/System.Private.CoreLib/src/System/Threading/NamedWaitHandleOptions.cs

```diff
@@ -0,0 +1,146 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+
+namespace System.Threading
+{
+    /// <summary>
+    /// Represents a set of options for named synchronization objects that are wait handles and can be shared between processes,
+    /// such as 'Mutex', 'Semaphore', and 'EventWaitHandle'.
+    /// </summary>
+    public struct NamedWaitHandleOptions
+    {
+        private bool _notCurrentUserOnly;
```

Comment:
I wonder what is the benefit of storing these as negative (_notXXX).

Reply:
We wanted a zero-initialized struct as in `default(NamedWaitHandleOptions)` to have `CurrentUserOnly == true` and `CurrentSessionOnly == true`, so the fields store the NOT of the property values.